src/util/fetchUtils.js

```diff
@@ -137,11 +137,10 @@ export const getFetchArgs = (apiUrl, queries, meta) => {
 		credentials: 'same-origin', // allow response to set-cookies
 	};
 	if (hasBody) {
-		config.body = isFormData
-			? queries[0].params
-			: fetchUrl.searchParams.toString();
+		config.body = isFormData ? queries[0].params : searchParams.toString();
 	}
-	const url = isFormData || !hasBody ? fetchUrl.toString() : apiUrl;
+	const url =
+		isFormData || apiUrl + (hasBody ? '' : `?${searchParams.toString()}`);
```

Comment:
watch out for shortcircuit here - `url` will be set to `true` when `isFormData` is true.

```js
const url = isFormData || !hasBody ? `${apiUrl}?${searchParams}` : apiUrl;
```